docs/modules/maintain-cluster/pages/rolling-upgrades.adoc

```diff
@@ -56,9 +56,9 @@ cluster version are always compatible, regardless of patch version.
 
 - Each minor version is compatible with the previous one.
 
-As a result, you can use rolling upgrades to upgrade clusters to the following member codebase versions:
+As a result, you can use rolling upgrades to upgrade clusters to the following member codebase versions within the same edition ({open-source-product-name} to {open-source-product-name} or {enterprise-product-name} to {enterprise-product-name} only):
 
-- A new patch version of the same major and minor version ({open-source-product-name} and {enterprise-product-name})
+- A new patch version of the same major and minor version ({open-source-product-name} or {enterprise-product-name})
```

Comment:
there are no patch version for OS published anymore, so mention about OS has no effect in practice